frontend/src/components/CreateCollectionModal/index.tsx

```diff
@@ -41,10 +40,7 @@ const CreateCollection: FC<{
   const [isOpen, setIsOpen] = useState(shouldModuleOpen);
```

Comment:
Turns out, this has been broken since #1016

Reply:
We need to define a new BOOLEAN enum. https://github.com/chanzuckerberg/corpora-data-portal/pull/762#pullrequestreview-546520583

Reply:
ahh great catch! I changed it to `href={`${API_URL}${API.LOG_IN}?redirect=?${QUERY_PARAMETERS.LOGIN_MODULE_REDIRECT}=${BOOLEAN.TRUE}`}`!